metadrive/obs/top_down_obs_multi_channel.py

```diff
@@ -256,13 +285,17 @@ def observe(self, vehicle: BaseVehicle):
             img = np.clip(img, 0, 255)
         return np.transpose(img, (1, 0, 2))
 
-    def draw_navigation(self, canvas, color=(128, 128, 128)):
+    def draw_navigation_node(self, canvas, color=(128, 128, 128)):
         checkpoints = self.target_vehicle.navigation.checkpoints
         for i, c in enumerate(checkpoints[:-1]):
             lanes = self.road_network.graph[c][checkpoints[i + 1]]
             for lane in lanes:
                 LaneGraphics.draw_drivable_area(lane, canvas, color=color)
 
+    def draw_navigation_trajectory(self, canvas, color=(128, 128, 128)):
+        lane = PointLane(self.target_vehicle.navigation.checkpoints, DEFAULT_TRAJECTORY_LANE_WIDTH)
```

Comment:
There is already an ego car trajectory in the simulator called `engine.map_manager.current_sdc_route`. But it is ok to use either of the two.

Reply:
Makes sense!